src/components/MyBooking/Mybooking.tsx

```
@@ -0,0 +1,98 @@
+  },
+];
+
+// Component to render individual service
+const ServiceCard = ({ service, onBook, isBusiness, onToggleActive }) => (
+  <Card
+    title={<Title level={4}>{service.name}</Title>}
+    bordered
+    style={{ borderRadius: '16px' }}
+    actions={[
+      isBusiness ? (
+        <Button type={service.active ? 'default' : 'primary'} onClick={() => onToggleActive(service.id)}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)